ppx/ppx_cstruct.ml

```diff
@@ -254,6 +439,10 @@ let output_hexdump _loc s =
                 [%expr Printf.bprintf _buf "0x%lx\n" ([%e Ast.evar (getter_name s f)] v)]
               |Prim UInt64 ->
                 [%expr Printf.bprintf _buf "0x%Lx\n" ([%e Ast.evar (getter_name s f)] v)]
+              |Bitfield (prim,len) ->
+                [%expr Printf.bprintf _buf
+                    (format_of_bitfield prim len)
```

Comment:
Fixed also hexdump to use valid format strings for bitfields, but not sure how to embed the function in here properly. PPX and AST documentation is really, really bad.